src/neptune/types/series/float_series.py

```diff
@@ -96,3 +103,16 @@ def unit(self):
 
     def __str__(self):
         return "FloatSeries({})".format(str(self.values))
+
+    def clear_of_unsupported_values(self):
+        cleared_values = []
```

Comment:
list comprehension + filter is probably faster. Use `append` in Python only in rare scenarios.

Reply:
Right! Changed that :D